ReleaseNotes

```diff
@@ -2905,7 +2907,7 @@ That command will mount the remote NFS server directory <remote-path> at
 the remote server.
 
 The NFS development was the graduation project of Jose Pablo Rojas V. who
-is a student at the Instituto Tecnol�gico de Costa Rica (ITCR, or TEC).
+is a student at the Instituto Tecnol�gico de Costa Rica (ITCR, or TEC).
```

Comment:
I see strange characters here introduced, maybe an encoding problem of the file while saving.

Reply:
These are in the original file they are unicode characters.

Reply:
ah I see my editor broke them further... Ugh, I'll see what I can do.

Reply:
I think that if you save it as UTF-8 it should be OK.

Reply:
By the way, look at this link: https://github.com/apache/incubator-nuttx/pull/1673/files#diff-20a19028a4ef22777368520a40d0cfdbL2908
The diff is displayed differently (the previous works looks OK) but it does not in this per-line comment.
I think you just found a bug in GitHub :laughing:

Reply:
Someone broke the unicode in the repo...  I am fixing it now.

Reply:
This is now fixed.  I was careful to recover all the names correctly using entries in the git log.